spec/controllers/event/qualifications_controller_spec.rb

```diff
@@ -49,6 +44,18 @@ def create_participation(role)
 
       it { expect(assigns(:participants).size).to eq(2) }
       it { expect(assigns(:leaders).size).to eq(1) }
+
+      context 'with qualifiable helpers' do
+        before do
+          Event::Role::Helper.qualifiable = true
+        end
+        after do
+          Event::Role::Helper.qualifiable = false
+        end
```

Comment:
Nicht wirklich falsch so, aber ich fände einen `around`-hook [1] hier etwas sprechender. Und es würde sogar ermöglichen, dass man leichter auf den vorherigen Wert statt `false` zurückwechselt. 

[1] https://relishapp.com/rspec/rspec-core/v/3-9/docs/hooks/around-hooks

```suggestion
        around(:example) do |example|
          prev_value = Event::Role::Helper.qualifiable
          Event::Role::Helper.qualifiable = true
          
          example.run
          
          Event::Role::Helper.qualifiable = prev_value
        end
```